feature-libs/order/components/order-history/order-history-extended/consignment-entries/consignment-entries.component.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component, ChangeDetectionStrategy, Input } from '@angular/core';
+import { Consignment } from '@spartacus/order/root';
+
+@Component({
+  selector: 'cx-consignment-entries',
+  templateUrl: './consignment-entries.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ConsignmentEntriesComponent {
+  @Input()
+  consignments?: Consignment[];
+  @Input()
+  orderCode?: string;
+
+  consignmentEntriesLength(consignment: Consignment): number {
+    if (consignment.entries) {
+      return consignment.entries.length;
+    }
+    return 0;
+  }
+
+  consignmentNumber(code?: string) {
```

Comment:
[MEIDUM]
Let's add an explicit return type to the method.

Reply:
done